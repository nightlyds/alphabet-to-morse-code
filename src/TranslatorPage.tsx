import React from "react";
import InputTextWrap from "./components/WrappedComponents/InputTextWrap";
import LanguagesWrap from "./components/WrappedComponents/LanguagesWrap";
import TranslatedTextWrap from "./components/WrappedComponents/TranslatedTextWrap";
import Description from "./components/Description";

const TranslatorPage: React.FC = () => {
    return (
        <div className="translator">
            <LanguagesWrap />
            <div className="translator-field">
                <InputTextWrap />
                <TranslatedTextWrap />
            </div>
            <Description />
        </div>
    );
};

export default TranslatorPage;
