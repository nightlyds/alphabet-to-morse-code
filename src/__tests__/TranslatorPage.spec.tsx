import React from "react";
import { shallow } from "enzyme";
import TranslatorPage from "../TranslatorPage";

let setUp = () => shallow(<TranslatorPage />);

describe("Translator tests", () => {
    describe("Snapshot tests", () => {
        let component: any;

        beforeEach(() => {
            component = setUp();
        });

        it("Basic snapshot test", () => {
            expect(component).toMatchSnapshot();
        });
    });
});
