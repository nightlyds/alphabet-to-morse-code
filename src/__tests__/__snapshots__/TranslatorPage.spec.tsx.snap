// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Translator tests Snapshot tests Basic snapshot test 1`] = `
<div
  className="translator"
>
  <Connect(Languages) />
  <div
    className="translator-field"
  >
    <Connect(InputText) />
    <Connect(TranslatedText) />
  </div>
  <Description />
</div>
`;
